qt/python/mantidqt/test/test_algorithm_dialog.py

```diff
@@ -0,0 +1,59 @@
+import unittest
```

Comment:
Missing GPL header.

Reply:
Also can we move the test to `mantidqt.widgets.test` with the others? I think this is also missing from the `python/CMakeLists.txt` file so that it is not being picked up by `ctest`.